includes/abstracts/class-evf-form-fields.php

```diff
@@ -176,7 +176,10 @@ public function field_preview( $field ) {}
 	 */
 	public function field_element( $option, $field, $args = array(), $echo = true ) {
 		$id     = (string) $field['id'];
-		$class  = ! empty( $args['class'] ) ? sanitize_html_class( $args['class'] ) : '';
+		//... Previous line of code
```

Comment:
Spacing before the comment. You can simply omit it out if the portion of code you commented out is deprecated. However, if you're documenting it for historical purposes, simply do something like:

```
// Deprecated <module> <version number> <reason>
// Deprecated codes goes here
```

@shivapoudel would love a point in the right direction here, as I've yet to come up against a circumstance like this.

Reply:
Lastly, you might want to run grunt tasks again to resolve the conflicts, or leave it up to code owners. :)

Reply:
Okay, thanks for the explanation.